.github/workflows/test-environment.yml

```diff
@@ -33,6 +29,10 @@ on:
         description: "Cleanup resources after provision"
         default: false
         type: boolean
+      ec-api-key:
+        type: string
+        description: "**Optional** By default, the environment will be created in our Cloud Security Organization. If you want to use your own cloud account, enter your Elastic Cloud API key."
+        required: true
```

Comment:
The same applies in this case. Despite its required: true setting, users must manually input values for the required inputs before the workflow can run, as specified in the [GitHub docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs). If we intend to treat it as optional, it should be interpreted as required: false. In that case, we would need to develop logic within an action (job step) to use the default API key defined as a secret when no API key is provided.